### PR TITLE
[Messenger] Add a note about allow_no_handlers option

### PR DESCRIPTION
**Pull request by Nek-**
This option was not documented at this state.

### CODE REVIEW

File: messenger/multiple_buses.rst [hunk 1]
```diff
@@ -105,6 +105,11 @@ This will create three new services:
 
 * ``event.bus``: autowireable with ``MessageBusInterface $eventBus``.
 
+.. note::
+
+    The ``allow_no_handlers`` option allows ``event.bus`` to have no and as a result,
```
Comment (noniagriconomie) [inline messenger/multiple_buses.rst]: Missing a word no?

File: messenger/multiple_buses.rst [hunk 2]
```diff
@@ -105,6 +105,11 @@ This will create three new services:
...
+    it will not fail if you defines no handler for this bus.
```
Comment (noniagriconomie) [inline messenger/multiple_buses.rst]: No s to define i think

**Comment by Nek-** [comment]
Thansk for fast review @noniagriconomie . I rephrased the thing!

**Comment by javiereguiluz** [comment]
Thank you Maxime! Please note that while merging we reworded this into a comment of the related configuration.

**Pull request merged**